src/classes/TDTM_Filter.cls

```diff
@@ -74,31 +74,31 @@ public class TDTM_Filter {
         filterObjectChain = new List<String>();
     }
     
-	/*******************************************************************************************************
+    /*******************************************************************************************************
     * @description Filters records to process.
     * @return FilteredLists An instance of the wrapper object that contains the filtered newList and oldList.
     */
     public FilteredLists filter() {
-    	try {
-    		filterField = tdtmToken.filterField;
-    		if(filterField != null) {
-	        	if(filterField.contains('.')) { //If the field to filter on is made of relationships
-	        	    filterByRelationship();
-	        	} else { //No need to break down the field condition
-	        	    if(filterField.endsWith('__c')) {
-	        	        fieldName = handleNamespace(filterField);
-	        	    } else {
-	        	        fieldName = filterField;
-	        	    }
-	        	    filterByField();
-	        	}
-	        	return filtered;
-	    	}
-    	} catch(Exception e) {
-    		UTIL_Debug.debug(LoggingLevel.WARN, '****Exception: ' + e.getMessage());
-    		UTIL_Debug.debug(LoggingLevel.WARN, '\n****Stack Trace:\n' + e.getStackTraceString() + '\n');
-    	}
-    	return null;
+        try {
+            filterField = tdtmToken.filterField;
+            if(filterField != null) {
+                if(filterField.contains('.')) { //If the field to filter on is made of relationships
```

Comment:
Side note, not a big fan of in line commenting. For consistency wise, should we move this line of comment above?

Reply:
@xinnanlisf I think this was a missed on my clean up part. I can remove it completely.